src/typeahead/typeahead.js

```diff
@@ -253,6 +255,10 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position'])
         function recalculatePosition() {
           scope.position = appendToBody ? $position.offset(element) : $position.position(element);
           scope.position.top += element.prop('offsetHeight');
+		            
+          scope.positionClearBtn =  $position.position(element);
+          scope.positionClearBtn.top += element.prop('offsetHeight')/4;
+          scope.positionClearBtn.left += element.prop('offsetWidth')-20; 		  
```

Comment:
For these two lines, add spacing for the math operations, i.e. `foo / 4` and `foo - 20`.